index.html

```diff
@@ -75,5 +75,251 @@ <h4>LETS CONNECT</h4>
                     </div>
               </div>
        </section> 
+       <section class="grid-container">
+             <div class="project-item">
+                <div class="project-image">
+                  <img src="./assets/first-image.jpg" alt="first project image"/>
+                </div>
+                <div class="project-details">
+                  <div>
+                    <h1>Tonic</h1>
```

Comment:
- [ ] Please don't use `h1` tags on titles that are not the main title of the webpage. This is for semantic reasons and HTML best practices. 🤓 This one can be an `h3`.